### PR TITLE
added geo-server helm charts

### PR DESCRIPTION
**Pull request by karthickp432001**

### CODE REVIEW

**Review by pranavv0 — COMMENTED** [review]
therr is no need to have .helmignore. its we already have .gitignore in main repo.

File: K8s-deployment/Charts/geo-server/install.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#! /bin/bash
+
+kubectl create namespace ugix-geoserver
```
Comment (pranavv0) [inline K8s-deployment/Charts/geo-server/install.sh:3]: do no not give name ugix in namespace ugix-geoserver, just geoserver is fine

Comment (karthickp432001) [reply K8s-deployment/Charts/geo-server/install.sh:3]: okay

Comment (karthickp432001) [reply K8s-deployment/Charts/geo-server/install.sh]: suggested changes have been updated ! please check

File: K8s-deployment/Charts/geo-server/README.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
+[![Kubescape Status](https://img.shields.io/jenkins/build?jobUrl=https%3A%2F%2Fjenkins.iudx.io%2Fjob%2Fkubescape-cat%2F&label=Kubescape)](https://jenkins.iudx.io/job/kubescape-cat/lastBuild/Kubescape_20Scan_20Report_20for_20CAT/)
```
Comment (karun-singh) [inline K8s-deployment/Charts/geo-server/README.md:1]: This is linked to kubescape pipeline for catalogue server/ Not needed as of now, please remove this.

File: K8s-deployment/Charts/geo-server/README.md [hunk 3]
```diff
@@ -0,0 +1,301 @@
...
+
+## Introduction
+
+Helm Chart for IUDX Geoserver Server Deployment
+
+## Create secret files
+
+1. Make a copy of sample secrets directory and add appropriate values to all files.
```
Comment (karun-singh) [inline K8s-deployment/Charts/geo-server/README.md:9]: There is no example-secrets directory. Please add that with the sample config.

File: K8s-deployment/Charts/geo-server/values.yaml
Comment (karun-singh) [inline K8s-deployment/Charts/geo-server/values.yaml:1]: Remove all the unnecessary verticle sections that are commented out currently. They are not required.

**Review by karun-singh — CHANGES_REQUESTED** [review]

**Comment by karthickp432001** [comment]
HI karun,

I have addded all the changes mentioned.

**Review by karun-singh — APPROVED** [review]
LGTM

**Pull request merged**